### PR TITLE
update sdk constructor

### PR DESCRIPTION
**Pull request by sanchaymittal**

### CODE REVIEW

File: packages/sdk/src/sdk.ts [hunk 1]
```diff
@@ -145,15 +146,20 @@ export class NxtpSdk {
...
     },
-    private readonly logger: Logger = new Logger({ name: "NxtpSdk", level: "info" }),
-    network: "testnet" | "mainnet" | "local" = "mainnet",
-    skipPolling = false,
   ) {
-    const { chainConfig, signer, messaging, natsUrl, authUrl } = this.config;
+    const { chainConfig, signer, messaging, natsUrl, authUrl, logger, network, skipPolling } = this.config;
+
+    this.logger = logger ?? new Logger({ name: "NxtpSdk", level: "info" });
+    this.config.network = network ?? "mainnet";
```
Comment (rhlsthrm) [inline packages/sdk/src/sdk.ts:160]: Let's default to testnet,

**Review by rhlsthrm — APPROVED** [review]
Changelog!

File: packages/sdk/src/sdk.ts (hunk 1 above)
Comment (sanchaymittal) [reply packages/sdk/src/sdk.ts:160]: okay!

File: packages/sdk/src/sdk.ts [hunk 2]
```diff
@@ -158,7 +158,7 @@ export class NxtpSdk {
     const { chainConfig, signer, messaging, natsUrl, authUrl, logger, network, skipPolling } = this.config;
 
     this.logger = logger ?? new Logger({ name: "NxtpSdk", level: "info" });
-    this.config.network = network ?? "mainnet";
+    this.config.network = network ?? "local";
```
Comment (rhlsthrm) [inline packages/sdk/src/sdk.ts:161]: Not local, "testnet".

**Review by rhlsthrm — CHANGES_REQUESTED** [review]

File: packages/sdk/src/sdk.ts (hunk 2 above)
Comment (sanchaymittal) [reply packages/sdk/src/sdk.ts:161]: ahh shoot my bad

**Review by rhlsthrm — APPROVED** [review]

**Pull request merged**